### PR TITLE
Refactor code 

### PR DESCRIPTION
**Pull request by AnastasiiaHanuliak**

### CODE REVIEW

**Review by AnastasiyaStr — APPROVED** [review]

File: src/main/java/com/softserve/demo/controller/LandingPageController.java [hunk 1]
```diff
@@ -32,8 +28,8 @@ public LandingPageController(ProviderFilter filter, FeedbackServiceImpl feedback
     }
 
     @GetMapping("providers/search-param")
-    public Page<?> getProvidersBySearchParam(@RequestParam(defaultValue = "0") int page,
-                                  @RequestParam int numberOfProvidersOnPage,
+    public Page<?> getProvidersBySearchParam(@RequestParam(defaultValue = Constant.ZERO) int page,
```
Comment (SilichAlex) [inline src/main/java/com/softserve/demo/controller/LandingPageController.java]: Generic "?" is a bad idea

Comment (AnastasiiaHanuliak) [reply src/main/java/com/softserve/demo/controller/LandingPageController.java]: I fixed it

**Review by AndriY2389 — APPROVED** [review]

File: src/main/java/com/softserve/demo/controller/LandingPageController.java (hunk 1 above)
Comment (AnastasiyaStr) [reply src/main/java/com/softserve/demo/controller/LandingPageController.java]: Bad ideas make best memories

File: src/main/java/com/softserve/demo/filter/ProviderFilter.java [hunk 2]
```diff
@@ -3,40 +3,39 @@
...
 import org.springframework.data.domain.PageRequest;
 import org.springframework.data.jpa.domain.Specification;
-import org.springframework.stereotype.Service;
-
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
 
-@Service
+@org.springframework.stereotype.Service
```
Comment (DimaNarepeha) [inline src/main/java/com/softserve/demo/filter/ProviderFilter.java:20]: Why to import in this way

Comment (AnastasiiaHanuliak) [reply src/main/java/com/softserve/demo/filter/ProviderFilter.java:20]: Because I have another Service from model package

Comment (DimaNarepeha) [reply src/main/java/com/softserve/demo/filter/ProviderFilter.java:20]: Oh, no problem

**Review by SilichAlex — APPROVED** [review]

**Pull request merged**